tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/PipesXUnitEvpTests.cs

```diff
@@ -0,0 +1,75 @@
+// <copyright file="PipesXUnitEvpTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+#if NETCOREAPP3_1_OR_GREATER
+using System;
+using System.Threading.Tasks;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.TestHelpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.CI;
+
+[Collection(nameof(TransportTestsCollection))]
+public class PipesXUnitEvpTests(ITestOutputHelper output) : XUnitEvpTests(output)
+{
+    [SkippableTheory]
+    [MemberData(nameof(GetData))]
+    [Trait("RunOnWindows", "True")]
+    [Trait("Category", "EndToEnd")]
+    [Trait("Category", "TestIntegrations")]
+    public override async Task SubmitTraces(string packageVersion, string evpVersionToRemove, bool expectedGzip)
+    {
+        SkipOn.Platform(SkipOn.PlatformValue.MacOs);
+        SkipOn.Platform(SkipOn.PlatformValue.Linux);
+        EnvironmentHelper.EnableWindowsNamedPipes();
+
+        // The server implementation of named pipes is flaky so have 5 attempts
+        var attemptsRemaining = 5;
```

Comment:
_5_ attempts? Is it really that flaky?! IIRC, looking at the metrics, we normally need 1 retry, tops 😬

Reply:
I saw a fail with 3 attempts, so I'm trying to play safe...

Reply:
Next week I'll look deeper on why the flakiness happens, but seems like something related to our mock server, the test being flaky do parallel connections and send approx 120 spans using those. Is very stable for small payloads.